content/en/quality_gates/guide/understanding_rule_scopes.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Understanding How Rule Scopes Work in Quality Gates
+kind: guide
+description: Learn how to configure rule scopes for Quality Gate rules.
+further_reading:
+  - link: "/quality_gates/setup"
+    tag: "Documentation"
+    text: "Learn how to set up Quality Gates"
+---
+
+## Overview
+
+Quality Gates allows you to gate your workflows based on signals in Datadog. When creating a rule, you can define a rule scope, which states when the rule should be evaluated. 
+
+To filter rules that are evaluated for a specific CI pipeline, you can add a custom scope when creating a rule. This process requires you to use the `--scope` option with the [`datadog-ci gate evaluate` command][1] in your build configuration.
+
+For example: 
+
+```shell
+datadog-ci gate evaluate --scope team:backend --scope team:frontend
+```
+
+## Define a rule scope
+
+When the `datadog-ci gate evaluate` command is invoked, the rules having a scope matching the command context are evaluated, and you can filter on rules that tag the `backend` or `frontend` teams.
+
+{{< img src="ci/rule_scope_always_evaluate.png" alt="Rule scope for rules always evaluated" style="width:80%;">}}
+
+For each scope (for example, `branch`), you can select included or excluded values.
+
+- When included values are selected, the rule is evaluated if one or more included values are part of the command context.
+- When excluded values are selected, the rule is not evaluated if any of the excluded values are part of the command context.
+
+To create a rule that is evaluated in all branches except `main` of the `example-repository` repository, you can create a rule with the following scope.
+
+1. Click `Select when to evaluate`. 
+1. Enter `example-repository` in the **Repository** field and click **Include**. 
+1. Click **Add Filter** and select **Branch**. 
+1. Enter `main` in the **Branch** field and click **Exclude**.
+
+{{< img src="ci/scope_not_main_example_repository.png" alt="Rule scope for example-repository and not main branch" style="width:90%;">}}
+
+If a rule does not contain a scope, it is evaluated for all values for that scope.
+For example, if a rule does not contain the `repository` scope, it is evaluated for all repositories.
+
+## Add a custom scope
```

Comment:
Just FYI, we're in the process of removing custom scopes from the product - it'll probably be removed in the next month; not sure if we want to remove it from the docs as we push this update out

Reply:
Ack, we can remove it then!